samples/quickstart.js

```diff
@@ -17,36 +17,33 @@
 
 // [START firestore_quickstart]
 const Firestore = require('@google-cloud/firestore');
-
-const firestore = new Firestore({
-  projectId: 'YOUR_PROJECT_ID',
-  keyFilename: '/path/to/keyfile.json',
-});
-
-const document = firestore.doc('posts/intro-to-firestore');
-
-// Enter new data into the document.
```

Comment:
Let's keep the comments that were describing what was happening

Reply:
Done